src/sustainable-web-design.test.js

```diff
@@ -113,4 +113,22 @@ describe("sustainable web design model", () => {
       });
     });
   });
+
+  describe("SWD Rating Scale", () => {
+    it("should return a string", () => {
+      expect(typeof swd.ratingScale(averageWebsiteInBytes)).toBe("string");
+    });
+
+    it("should return a rating", () => {
+      // Check a 3MB file size
+      expect(swd.ratingScale(3000000)).toBe("F");
+    });
+
+    it("returns ratings as expected", () => {
+      expect(swd.ratingScale(0.065)).toBe("A+");
```

Comment:
I think using the constants here would make the test slightly clearer, as you don't need to look up the significance of 0.65, and we only have one place to make a change when we are using data from a new HTTP Archive snapsho